models/base_model.py

```diff
@@ -11,10 +11,21 @@
 
 class BaseModel:
     """Initialize a BaseModel class"""
-    def __init__(self):
-        self.id = str(uuid.uuid4())
-        self.created_at = datetime.datetime.now()
-        self.updated_at = datetime.datetime.now()
+    def __init__(self, *args, **kwargs):
+        if kwargs is not None and kwargs != {}:
+            for key in kwargs:
+                if key == "created_at":
+                    self.__dict__["created_at"] = datetime.strptime(
+                            kwargs["created_at"], "%Y-%m-%dT%H:%M:%S.%f")
+                elif key == "updated_at":
+                    self.__dict__["updated_at"] = datetime.strptime(
+                            kwargs["updtaed_at"], "%Y-%m-%dT%H:%M:%S.%f")
```

Comment:
You have a typo in the variable name. It should be "updated_at" instead of "updtaed_at"